src/modules/daemonconfiguration/src/daemonconfiguration.rs

```diff
@@ -83,7 +83,10 @@ impl SystemctlInfo for Systemctl {
                 continue;
             }
             let daemon = Self::create_daemon(&service["service"], &service["status"])?;
-            services.push(daemon);
+            // Only report enabled daemons with State not being Other
+            if daemon.state != State::Other && daemon.state != State::Dead && daemon.auto_start_status == AutoStartStatus::Enabled {
```

Comment:
>f daemon.state != State::Other && daemon.state != State::Dead && daemon.auto_start_status == AutoStartStatus::Enabled  [](http://example.com/codeflow?start=13&length=118)

Does Rust have operator precedence? Then, don't rely on it and separate statement with extra parenthesis?

Reply:
Yes, however adding extra parentheses often results in linting errors during `cargo build` which will show up as warnings when osconfig is build via `cmake`. @9KelvinN I suggest adding parentheses wherever possible without creating build warnings like @MariusNi suggests (we should not suppress these types of warnings).

Reply:
Also, the opposite is true: the compiler will warn you if the precedence between 2 operators is unclear and suggest adding parenthesis where necessary to avoid such operator precedence errors.

https://doc.rust-lang.org/reference/expressions.html

Reply:
Heeded the suggestions, added parentheses to explicitize order of operations